### ISSUE TITLE
[Update term] `Continuous Delivery` for `Hindi`

### ISSUE THREAD

**Issue by jihoon-seo** [cncf/glossary#667]
#### Why this term needs to be updated? (briefly) 

Current preview: https://deploy-preview-665--cncfglossary.netlify.app/hi/continuous_delivery/

In that page, it would be hard to know that `CD` is the English abbreviation of `Continuous Delivery`.

My suggestion:

```diff
-title: निरंतर डिलीवरी (CD)
+title: निरंतर डिलीवरी (Continuous Delivery)
```

Plus, if needed, we might state that `CI` is the English abbreviation of `Continuous Integration`, and `PR` is of `Pull Request`.

```diff
-निरंतर इंटीग्रेशन (CI)
# One can choose one of these styles:
+निरंतर इंटीग्रेशन (CI, Continuous Integration)
+निरंतर इंटीग्रेशन (CI; Continuous Integration)
+निरंतर इंटीग्रेशन (Continuous Integration, CI)
+निरंतर इंटीग्रेशन (Continuous Integration; CI)
```

```diff
-पुल रिक्वेस्ट (PR)
# One can choose one of those styles:
+-पुल रिक्वेस्ट (PR; Pull Request)
```

For the last one, we might update the link to the English version of `Canary Deployment`, to the [Hindi version](https://deploy-preview-665--cncfglossary.netlify.app/hi/canary_deployment/).
```diff
-[कैनरी](/canary_deployment/)
+[कैनरी](/hi/canary_deployment/)
```

PTAL Hindi contributors! 😊

**Comment by Shivansh-yadav13** [comment]
Can I add them?

**Comment by jihoon-seo** [comment]
I think that it will be good to take the triage process for this issue.

Dear @Shivansh-yadav13 and other Hindi contributors,
if you think that my suggestion makes sense and seems necessary, then please let me know, and I will update the triage status to `accepted`. 😊

**Comment by anubha-v-ardhan** [comment]
Thanks for pointing these out @jihoon-seo 

I've assigned you @Shivansh-yadav13. Thankyou for showing interest

**Comment by Shivansh-yadav13** [comment]
@jihoon-seo we can close this now?

**Comment by anubha-v-ardhan** [comment]
fixed with #681
